### DISCUSSION TITLE
Host stops as soon as I enable pass through on the VM

### DISCUSSION THREAD

**Discussion by shivamgrover21** [isc30/ryzen-gpu-passthrough-proxmox#11] [General]
Hi everyone, I completed these steps on um790 pro. But after configuring the VM for passthrough and turning it on, the host screen goes black and proxmox webui is not reachable anymore. 

I’m using the seabios and if i restart the device, it gets stuck on initiating ramdisk.

I think I’m making a mistake while loading the vfio-pci in the VM configuration in the file 
/etc/modprobe.d/vfio.conf.

Could someone please share their configuration on how this looks like ?

I’ll share my configuration in a while that i did because I’ll have to reinstall everything and i cannot get the host to turn on anymore.

**Comment by mr-suw** [comment]
Hi @shivamgrover21,

which Proxmox release version are you running?
I had pass-through running with AMD Ryzen 9 7940HS on Proxmox version 8.0, but it seems it is not working on 8.1 yet.

`cat /etc/modprobe.d/vfio.conf`
options vfio-pci ids=1002:15bf,1002:1640,144d:a80a
softdep radeon pre: vfio-pci
softdep amdgpu pre: vfio-pci
softdep snd_hda_intel pre: vfio-pci

`lspci -kkn`
c8:00.0 0300: 1002:15bf (rev c1)
   Subsystem: 1f66:0031
   Kernel driver in use: vfio-pci
   Kernel modules: amdgpu
c8:00.1 0403: 1002:1640
   Subsystem: 1002:1640
   Kernel driver in use: vfio-pci
   Kernel modules: snd_hda_intel

**Reply by shivamgrover21** [reply]
Thanks so much for your reply mate. I went ahead directly with proxmox 8.1 ve.

I’ll try today again with 8.0. Hopefully it works.

**Reply by shivamgrover21** [reply]
you're absolutely brilliant. it does work with proxmox 8.0
maybe you could help me with another thing. I had installed gui xfce4vlightdm on the proxmox host so that I can directly connect the host machine to my screen 
But as soon as I start the windows vm with igpu passthough, understandably, the host goes blank.
Is there a way that i can still access the host machine via gui directly and also keep the windows vm turned on with passthrough ?

**Reply by isc30** [reply]
Unless you passthrough the /dev/dri/ devices directly to LXC, I don't think it's possible to have both at the same time (it's either the host uses the gpu for vnc or the vm does it)

**Reply by shivamgrover21** [reply]
alright, thanks mate
